doc/scripts/GMT_App_M_1d.sh

```diff
@@ -40,28 +41,20 @@ do
 	j=$(expr $i + 1)
 	left=$(sed -n ${j}p tt.lis)
 	right=$(sed -n ${i}p tt.lis)
-	gmt makecpt -H -C$left -T-1/1 > tt.left.cpt
-	gmt makecpt -H -C$left -T-1/1/0.25 > tt.left2.cpt
+	if [ -n "$left" ]; then
+	  gmt makecpt -H -C$left -T-1/1 > tt.left.cpt
+	  gmt makecpt -H -C$left -T-1/1/0.25 > tt.left2.cpt
+	fi
 	gmt makecpt -H -C$right -T-1/1 > tt.right.cpt
 	gmt makecpt -H -C$right -T-1/1/0.25 > tt.right2.cpt
-	gmt colorbar -Dx1.55i/${y}i+w2.70i/0.125i+h+jTC -Ctt.left.cpt -B0
+	if [ -n "$left" ]; then
+	  gmt colorbar -Dx1.55i/${y}i+w2.70i/0.125i+h+jTC -Ctt.left.cpt -B0
+	  gmt colorbar -Dx1.55i/${y2}i+w2.70i/0.125i+h+jTC -Ctt.left2.cpt -Bf0.25
+		echo 1.55 $y ${left} | gmt text -D0/0.05i -F+f9p,Helvetica-Bold+jBC
+	fi
 	gmt colorbar -Dx4.50i/${y}i+w2.70i/0.125i+h+jTC -Ctt.right.cpt -B0
```

Comment:
```suggestion
	if [ ${i} == ${n2} ] && [ $((i % 2)) != 0 ]; then
		x=3.05i
	else
		x=4.50i
	fi
	gmt colorbar -Dx${x}/${y}i+w2.70i/0.125i+h+jTC -Ctt.right.cpt -B0
```

Reply:
This centers the top panels if there are a odd number of colormaps

Reply:
Nice, I've tweaked the script a little bit in e42d67312c0a8fd8c1842f4650e9cb224628773f so that the 'vikO' text label is centered as well. Now it looks like this:

![GMT_App_M_1d.png](https://user-images.githubusercontent.com/23487320/108776261-0ce1d300-75c7-11eb-9ec3-adf1dfba1f1d.png)